docs/bundler.md

```diff
@@ -78,8 +78,19 @@ The directory that Bundler will look into when installing gems.
 
 ### BUNDLE_DEPLOYMENT
 
-Disallow changes to the **Gemfile**. When the **Gemfile** is changed
-and the lockfile has not been updated, running Bundler commands will be blocked.
+Disallow changes to the **Gemfile**.
```

Comment:
```diff
- Disallow changes to the **Gemfile**.
+ Disallow changes to the **Gemfile**. When the **Gemfile** is changed
```
`When the **Gemfile** is changed` went missing.

Reply:
Ok, because with the change as it is now the next sentence starts from the middle: `and the lockfile has not been updated, running Bundler commands will be blocked.`

Reply:
Adjusted, although a bit differently than suggested @a-ovchinnikov. Merely cutting and pasting the suggested content to a location earlier in the document didn't feel (contextually) right to me and so I reworded the paragraph slighly and added a `Offline installation pitfalls` section to host it, let me know if you agree.

Reply:
Derp, sorry, I thought you were referring to the other occurrence in  the design doc where there's only the first sentence (pre-existing) and because I was editing mainly that one I thought I omitted something on purpose. Sorry, for the confusion, this is now fixed!